docs/querying/sql-functions.md

```diff
@@ -1133,11 +1152,26 @@ Returns the minimum value of a set of values.
 
 ## MOD
 
-`MOD(x, y)`
+Calculates x modulo y, or the remainder of x divided by y. Where x and y are numeric expressions.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `MOD(x, y)`
+* **Function type:** Scalar, numeric
+
+<details><summary>Example</summary>
+
+The following calculates 78 MOD 10.
+
+```sql
+SELECT MOD(78, 10) as "mod"
```

Comment:
nit: suggest using an english word for the alias (kind of the point of an alias :D )